README.md

```diff
@@ -1,55 +1,37 @@
 # iRODS TypeScript HTTP API Wrapper
 
-## Installation and Usage
-
-1. Install npm package: 
+## Installation
 
 ```
 npm install git+https://github.com/irods/irods_client_http_typescript
 ```
 
-2. Import `IrodsHttpClient` class:
+## Usage
+
+After creating an instance of the wrapper, you can authenticate it within an asynchronous context, then do any operations available from the http api. The following code demonstrates a way of creating a new collection with the wrapper.
 
 ```js
 import { IrodsHttpClient } from 'irods_client_http_typescript'
-```
+import type { URLComponentsType } from 'irods_client_http_typescript'
 
-3. Create a url configuration object, containing info about the irods http api that's running on your local machine:
-
-```js
+// Create a url configuration object, containing info about
+// the irods http api that's running on your local machine.
 const urlComponents: URLComponentsType = {
     host: '<host>',
     port: '<port>',
     version: '<version>',
 }
-```
 
-4. Create a wrapper object, passing in the url components, as well as the username and password you want to log in with:
-```js
+// Create an IrodsHttpClient object
 const api = new IrodsHttpClient(urlComponents, '<username>', '<password>')
-```
 
-5. Authenticate the wrapper object within an asynchronous context
+let res
+let lpath = "/tempZone/home/<username>/newCollection"
 
-```js
+// Authenticate the client within an asynchronous context.
+// From there, any operations of the api can be done by the client.
 (async () => {
     await api.authenticate()
-})
-```
-
-
-- **Note**: This code is assuming you're not already within an asynchronous context, such as an async function. 
-
-6. All operations of the wrapper can now be done within an asynchronous context as well, e.g.,
-
-```js
-import { CollectionTypes } from "irods-http-api-wrapper";
-
-// Declare type of res if working in typescript, otherwise just put 'let res'
-let res: CollectionTypes.CollectionCreateResponse | null
-let lpath = "/tempZone/home/user/newCollection"
-
-(async () => {
     res = await api.collections.create({
         lpath: lpath
     })
```

Comment:
The example needs to demonstrate how to handle errors too.

That is, show how to check the HTTP status code and then the iRODS status code.

Reply:
Done.